x.payment_batching/payment_batching.md

```diff
@@ -0,0 +1,307 @@
+In that way, increased use of payment batching may lower the feerate for
+all Bitcoin users.
+
+In summary, payment batching provides significant savings for services
+that typically have inputs available that are 5 to 20 times larger than
+their typical output.  For services not in that position, the savings
+from batching alone are smaller but perhaps still worth the effort;
+if the services are willing to also pre-consolidate their inputs, the
+savings can be quite dramatic.
+
+Note: the figures and plots above all assume use of P2WPKH inputs and
+outputs.  We expect that to become the dominant script type on the
+network in the future (until something better comes along).  However, if
+you use a different script type (P2PKH, or multisig using P2SH or
+P2WSH), the number of vbytes used to spend them are even larger, so the
+savings rate will be higher.
+
+## Problems
+
+The fee-reduction benefits of payment batching do create tradeoffs and
+problems that will need to be addressed by any service using the
+technique.
+
+### Delayed gratification
+
+This is the primary problem with payment batching.  Although some
+situations naturally lend themselves to payment batching (e.g.  sending
+out the company payroll for every employee at the same time), many
+services primarily send money to users when those users make a
+withdrawal request.  In order to batch payments, the service must get
+the user to accept that their payment will not be sent immediately---it
+will be held for some period of time and then combined with other
+withdrawal requests.
+
+One mitigation for the problem of delayed gratification is to allow the
+user to choose between an immediate payment and a delayed payment, with
+a different fee provided for each option.  For example:
+
+    [X] Free withdrawal (payment sent within 6 hours)
+    [ ] Immediate withdrawal (withdrawal fee 0.123 mBTC)
+
+Another mitigation is to use transaction replacement to append new
+outputs to an existing transaction.  This allows you to send each payment
+immediately, but each replacement must pay a higher feerate than the
+earlier version of the transaction.  This reduces your savings rate
+unless you were planning to fee bump an earlier transaction anyway.
+We'll cover batching and replacement in more detail later in this
+chapter.
+
+## Reduced privacy
+
+A second problem with payment batching is that it can make users feel
+like they have less privacy.  Every user you pay in the same transaction
+can reasonably assume that everyone else receiving an output from that
+transaction is being paid by you.  If you had sent separate
+transactions, any onchain relationship between the payments might be
+less apparent or even non-existent.
+
+![Screenshot of a possible transaction batch in a block explorer](img/batch-screenshot.png)
+
+Note that transactions belonging to particular Bitcoin services are
+often identifiable by experts even if they don't use payment
```

Comment:
> it's trivially easy to correlate blockchain usage. See oxt.me for example.

OXT looks amazing, but have you actually tested it?  E.g. looked up transactions you know came from or went to a particular service (or didn't) and counted the actual correct results, failed identifications, and misidentifications?  I'm not trying to impugn their reputation, I'm just wondering whether you've tested your belief in their capability.  (I haven't tested it myself, so I have no idea how well it compares to an analysis by an expert.)

Reply:
Thanks for calling me out on this. No, I haven't done any testing of OXT myself. I've just assumed that the Laurent's analysis is good.

Incidentally, he's got a good blog here: https://medium.com/@laurentmt . It'd be interesting to see if he's written anything about his methodology in exchange tx correlation.

Reply:
So I looked up about a dozen transactions I made via Coinbase (2016, 2017, and 2019), both sends and receives, and OXT didn't identify a single one of them as involving Coinbase (although it did successfully identify my 2016/2017 wallet[1]).  I then tried the same search on https://www.walletexplorer.com and it also easily identified my 2016/17 wallet; I was also able to do a bit extra legwork there to find what looked like Coinbase's wallet a few hops from my transactions (but it was not labeled as such in the site).

The conclusion I'd draw from this small sample is that the current text is probably about right in mentioning the use of an expert and that batching would probably increase the ease of correlation for transactions from high-frequency spenders.

[1] Probably because I sold all my Bcash forkcoins in a single tx.